packages/migrate/src/utils/getDatabaseVersionSafe.ts

```diff
@@ -1,14 +1,15 @@
-import Debug from '@prisma/debug'
+// import Debug from '@prisma/debug'
 
-import { MigrateEngine } from '../MigrateEngine'
+// import { MigrateEngine } from '../MigrateEngine'
 
-const debug = Debug('prisma:cli')
+// const debug = Debug('prisma:cli')
```

Comment:
Intentional?

Reply:
Yes, code in this module is commented and TODO says to uncomment it when https://github.com/prisma/prisma-private/issues/203 is fixed. Imports were still there and were marked as unused by the rule.